lib/matplotlib/axes/_axes.py

```diff
@@ -7811,14 +7811,15 @@ def violinplot(self, dataset, positions=None, vert=True, widths=0.5,
         Make a violin plot.
 
         Make a violin plot for each column of *dataset* or each vector in
-        sequence *dataset*.  Each filled area extends to represent the
-        entire data range, with optional lines at the mean, the median,
+        sequence *dataset*. Each filled area extends to represent the entire data range, with optional lines at the mean, the median,
         the minimum, and the maximum.
 
         Parameters
         ----------
-        dataset : Array or a sequence of vectors.
-          The input data.
+        dataset : 2D Array/Matrix or a sequence of vectors.
+          The input data. 
```

Comment:
don't need "the input data".  Notes about data haven't been moved to notes section"